druid-shell/src/platform/mac/clipboard.rs

```diff
@@ -34,6 +34,7 @@ impl Clipboard {
             unsafe {
                 let nsstring = util::make_nsstring(s);
                 let pasteboard: id = msg_send![class!(NSPasteboard), generalPasteboard];
+                let _: NSInteger = msg_send![pasteboard, clearContents];
```

Comment:
it might be worth making this a separate patch? it's a bug unrelated to this PR, and its fate isn't tied to this work.